packages/governance/src/contractHelper.js

```diff
@@ -17,7 +17,7 @@ const { details: X, quote: q } = assert;
  *  parameter values, and the governance guarantees only hold if they're not
  *  used directly by the governed contract.
  *
- * @param {ContractFacet} zcf
+ * @param {ZoeCF} zcf
```

Comment:
I'm not sure `ZoeCF` is a better name.  Adding "Zoe" is helpful, but we use `CF` in a lot of places here and in governance to mean `CreatorFacet`.

Reply:
I think that works quite well.

Reply:
4a7bcc